build/yamls/windows/ovs-containerd/ovs-containerd.yml

```diff
@@ -0,0 +1,110 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    component: antrea-agent
+  name: antrea-agent-windows
+spec:
+  selector:
+    matchLabels:
+      component: antrea-agent
+  template:
+    metadata:
+      annotations:
+        "microsoft.com/hostprocess-inherit-user": "true"
+      labels:
+        component: antrea-agent
+    spec:
+      securityContext:
+        windowsOptions:
+          runAsUserName: "NT AUTHORITY\\SYSTEM"
+          hostProcess: true
+      containers:
+        - name: antrea-agent
+          image: antrea-windows
+          imagePullPolicy: IfNotPresent
+          lifecycle:
+            preStop:
+              exec:
+                command: ["powershell", "Disable-VMSwitchExtension -Name 'Open vSwitch Extension' -VMSwitchName antrea-hnsnetwork"]
```

Comment:
Will it fail if agent crashes before creating HNSNetwork (meaning if the HNSNetwork with name antrea-hnsnetwork does not exist)?

Reply:
I think It will show preStopHook failure and try to restart, then agent will try to recreate hnsnetwork on host.